client/frontend/template/retry

```diff
@@ -0,0 +1,70 @@
+import (
+	"context"
+
+	"go.uber.org/yarpc"
+
+	"github.com/uber/cadence/common/backoff"
+	"github.com/uber/cadence/common/types"
+)
+
+{{ $decorator := (printf "retryable%s" .Interface.Name) }}
+{{ $Decorator := (printf "Retryable%s" .Interface.Name) }}
+
+var _ {{.Interface.Name}} = (*{{$decorator}})(nil)
+
+// {{$decorator}} implements {{.Interface.Type}} interface instrumented with retries
+type {{$decorator}} struct {
+    client        {{.Interface.Type}}
+    throttleRetry *backoff.ThrottleRetry
+}
+
+// New{{$Decorator}} creates a new instance of {{$decorator}} with retry policy
+func New{{$Decorator}}(
+    client {{.Interface.Name}},
+    policy backoff.RetryPolicy,
+    isRetryable backoff.IsRetryable,
+) {{.Interface.Name}} {
+    return &{{$decorator}}{
+        client: client,
+        throttleRetry: backoff.NewThrottleRetry(
+            backoff.WithRetryPolicy(policy),
+            backoff.WithRetryableError(isRetryable),
+        ),
+    }
+}
+
+{{range $method := .Interface.Methods}}
+    {{if or
+        (eq $method.Name "CountWorkflowExecutions")
+        (eq $method.Name "GetSearchAttributes")
+        (eq $method.Name "RefreshWorkflowTasks")
```

Comment:
alrighty, from a manual check: yea, only these 3 changed, now no longer do retries.

do we not want retries on these for some reason though?  or are these maybe just leftovers from developing the template?

Reply:
There used to have exceptions but now it seams they are all gone.

Reply:
yea.  can't really see why these would be special tbh.  seems harmless to redo them on failures.